scripts/navigation.js

```diff
@@ -0,0 +1,5 @@
+function showMenu() {
+  document
+    .getElementById("navigationList")
+    .classList.toggle("navigation__list-mobile--active");
+}
```

Comment:
This is all okay - but the function name does not represent what the function does. And why not give the function and the file the same name for a better discovery?